hosts/hetzner-ephemeral-alpha/default.nix

```diff
@@ -13,12 +13,6 @@
   networking.hostName = "hetzner-ephemeral-alpha";
   time.timeZone = "Europe/Helsinki";
 
-  # Support for cross compilation
-  boot.binfmt.emulatedSystems = [
-    "aarch64-linux"
```

Comment:
Given that we might need to cross-build the aarch64-linux on x86 anyway, I think this should stay enabled after all. 32bit arm should stay disabled.